README.md

```diff
@@ -339,6 +343,11 @@ Now, we can create our virtual environment by typing the following (note the `.`
 ```
 $ virtualenv --no-site-packages .
 ```
+*As of July 27 2015, --no-site-packages is default behavior the flag is deprecated. Run with:
+
+```
+$ virtualenv .
+```
```

Comment:
Good catch!  I think it would be more clear to put this in a blockquote:

-------

 ```
 $ virtualenv --no-site-packages .
 ```
> *Note: As of July 27 2015, --no-site-packages is default behavior the flag is deprecated. Run with:*
> 
> ```
> $ virtualenv .
> ```